### DISCUSSION TITLE
Set Wifipower before compiling

### DISCUSSION THREAD

**Discussion by classiphil** [arendst/Tasmota#11620] [Q&A] [answered]
Hello there,

I really (really) love the new Command to set the TX Power of the Wifi. Now I want to set it smaller than 17 before compiling, so that I don´t have to set it via the command after flashing.

I don´t find the setting in the configuration file.

Thanks for your help.

**Comment by sfromis** [comment, marked as answer]
There is no `user_config_override.h` setting for that (not a new command). The default `wifi_output_power` is hardcoded [here](https://github.com/arendst/Tasmota/blob/development/tasmota/settings.ino#L789). You could also have it in a boot rule or an `autoexec.bat` file (if on a board with more than 1MB flash).

**Comment by classiphil** [comment]
Thank you sfromis,

so could I just set this value to 50 before compiling and have a WIFIPower of 5?

**Reply by sfromis** [reply]
That matches the [WifiPower](https://github.com/arendst/Tasmota/blob/development/tasmota/support_command.ino#L2090-L2100) command.
